recipes/orca-test/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "orca_test" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/UDST/orca_test/archive/v{{ version }}.tar.gz
+  sha256: 3107b31578b2f6e45c5e57b8bc45c94bf2d59f3d3ad1d7162bf36537f7209cca
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.0
+    - orca >=1.3.0
+    - pandas >=0.12
+    - python
+
+test:
+  imports:
+    - orca_test
+
+about:
+  home: https://github.com/UDST/orca_test
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: Assertions for testing the characteristics of orca tables and columns
+
+extra:
+  recipe-maintainers:
+    - smmaurer
```

Comment:
@chrisburr After chatting with **UDST** members, we decided to remove **UDST** organization as a maintainer and  add `smmaurer` instead. @smmaurer can confirm this :)